resources/js/Pages/Deals/Components/styles.css

```
@@ -1,3 +1,5 @@
+@reference "../../../../css/tailwind.css";
```

Stylelint (17.13.0)

[error] Unexpected unknown at-rule "`@reference`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)